.ci/gcb-generate-diffs-new.yml

```diff
@@ -64,6 +64,17 @@ steps:
           - --no-ff
           - head/$_HEAD_BRANCH
 
+    - name: 'gcr.io/graphite-docker-images/pre-build-validator'
+      id: pre-build-validate
+      secretEnv: ["GITHUB_TOKEN"]
+      waitFor: ["merged"]
+      args:
+          - $_PR_NUMBER
+          - $COMMIT_SHA
```

Comment:
args shouldn't be needed. Lets use env instead. args were only retained in the other instance to preserve the behavior for current prs calling that workflow that wouldn't have env present in the generate-diffs yaml

Reply:
Done.